qabelbox/src/androidTest/java/de/qabel/android/util/PersistenceTest.java

```diff
@@ -0,0 +1,142 @@
+package de.qabel.android.util;
+
+
+import android.test.AndroidTestCase;
+
+import junit.framework.Assert;
+
+import java.io.Serializable;
+import java.util.List;
+
+import de.qabel.core.config.Persistable;
+import de.qabel.android.config.AndroidPersistence;
+import de.qabel.android.config.QblSQLiteParams;
+
+public class PersistenceTest extends AndroidTestCase {
+    private final static String DB_NAME = "qabel-android-test";
+    private final static int DB_VERSION = 1;
+    private AndroidPersistence persistence;
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+        getContext().deleteDatabase(DB_NAME);
+        QblSQLiteParams params = new QblSQLiteParams(getContext(), DB_NAME, null, DB_VERSION);
+        persistence = new AndroidPersistence(params);
+    }
+
+    @Override
+    public void tearDown() throws Exception {
+        super.tearDown();
+    }
+
+    public void testGetNotPersistedEntity() {
+        Assert.assertNull(persistence.getEntity("1", PersistenceTestObject.class));
```

Comment:
`import static Assert.*` (or specific) to make the tests smaller. Then you can just use
`assertNull(...)`